internal/data/receivers_wallet.go

```diff
@@ -616,3 +576,115 @@ func (rw *ReceiverWalletModel) UpdateInvitationSentAt(ctx context.Context, sqlEx
 
 	return receiverWallets, nil
 }
+
+type ReceiverWalletUpdate struct {
+	Status                      *ReceiversWalletStatus `db:"status"`
+	AnchorPlatformTransactionID *string                `db:"anchor_platform_transaction_id"`
+	StellarAddress              *string                `db:"stellar_address"`
+	StellarMemo                 *string                `db:"stellar_memo"`
+	StellarMemoType             *string                `db:"stellar_memo_type"`
+	OTPConfirmedAt              *time.Time             `db:"otp_confirmed_at"`
+	OTPConfirmedWith            *string                `db:"otp_confirmed_with"`
+}
+
+func (rwu ReceiverWalletUpdate) IsEmpty() bool {
+	return rwu.Status == nil &&
+		rwu.AnchorPlatformTransactionID == nil &&
+		rwu.StellarAddress == nil &&
+		rwu.StellarMemo == nil &&
+		rwu.StellarMemoType == nil &&
+		rwu.OTPConfirmedAt == nil &&
+		rwu.OTPConfirmedWith == nil
+}
```

Comment:
Since you're testing if LL fields are empty you should be able to replace this method with `utils.IsEmpty`: 
https://github.com/stellar/stellar-disbursement-platform-backend/blob/74da14334a3f7c837e50a874d9a3b4fcce197dd2/internal/utils/utils.go#L45-L53

Reply:
Addressed in c174a217289def231177fad9bcfc56475f2200e7